package.json

```
@@ -17,10 +18,14 @@
     "filesize": "^10.1.2"
   },
   "devDependencies": {
+    "@commitlint/cli": "^20.1.0",
+    "@commitlint/config-conventional": "^20.0.0",
+    "@ffmpeg-installer/ffmpeg": "^1.1.0",
     "electron": "^31.0.0",
     "electron-builder": "^24.13.3",
-    "@ffmpeg-installer/ffmpeg": "^1.1.0",
-    "@ffprobe-installer/ffprobe": "^2.1.3"
+    "@ffprobe-installer/ffprobe": "^2.1.3",
```

```suggestion
    "@ffmpeg-installer/ffmpeg": "^1.1.0",
    "@ffprobe-installer/ffprobe": "^2.1.3",
```
any specific reason why this was removed? If it was accidental, lets bring it back.